### PR TITLE
feat(ts): Add better tests for typescript

### PR DESCRIPTION
**Pull request by Vorobeyko**
## Motivation/Description of the PR
- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [ ] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: typings/tests/global-variables.types.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  callback // $ExpectType HookCallback
+)
+Scenario('scenario',
+  (args) => {
+    args // $ExpectType SupportObject
+    args.I // $ExpectType I
+  }
+)
+
+// Before((
```
Comment (pablopaul) [inline typings/tests/global-variables.types.ts:32]: @Vorobeyko please remove commented code

Comment (Vorobeyko) [reply typings/tests/global-variables.types.ts:32]: Its draft PR) I'm still working on it

Comment (pablopaul) [reply typings/tests/global-variables.types.ts:32]: @Vorobeyko ok sorry, I just saw it now :/

File: .eslintrc.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "no-undef": 0,
     "prefer-destructuring": 0,
     "no-param-reassign": 0,
+    "no-multiple-empty-lines": 0,
```
Comment (Vorobeyko) [inline .eslintrc.json:14]: need remove

File: docs/webapi/grabElementBoundingRect.mustache [hunk 3]
```diff
@@ -15,6 +15,6 @@ To get only one metric use second parameter:
 const width = await I.grabElementBoundingRect('h3', 'width');
 // width == 527
 ```
-@param {string|object} locator element located by CSS|XPath|strict locator.
-@param {string} elementSize x, y, width or height of the given element.
-@returns {object} Element bounding rectangle
\ No newline at end of file
+@param {LocatorOrString} locator element located by CSS|XPath|strict locator.
+@param {string=} elementSize x, y, width or height of the given element.
```
Comment (PeterNgTr) [inline docs/webapi/grabElementBoundingRect.mustache:19]: just curious why we had `=` after?

Comment (Vorobeyko) [reply docs/webapi/grabElementBoundingRect.mustache:19]: this syntax uses for optional params

**Pull request merged**